### PR TITLE
Add Gardener project to platform-setup

### PR DESCRIPTION
**Pull request by loewenstein**

### CODE REVIEW

**Comment by loewenstein** [comment]
cc @vasu1124 in case of questions regarding Gardener.

**Comment by geeknoid** [comment]
There are a few CI failures to address:

./content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md:9: MD022 Headers should be surrounded by blank lines
./content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md:23: MD029 Ordered list item prefix
./content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md:30: MD029 Ordered list item prefix

File: content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Gardener - Managed Kubernetes clusters
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md]: How about just calling it "Kubernetes Gardener"?

File: content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Bootstrapping Gardener
+To set up your own [Gardener](https://gardener.cloud), see the
+[documentation](https://github.com/gardener/gardener/blob/master/docs/README.md)
+or have a look at the
+[landscape-setup-template](https://github.com/gardener/landscape-setup-template)
+project. To learn more about this open source project, read the
+[blog on kubernetes.io](https://kubernetes.io/blog/2018/05/17/gardener/).
+
+### Install and configure kubectl
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md]: Put kubectl in backticks

**Comment by loewenstein** [comment]
@geeknoid This is no less Istio specific than the sibling pages about GKE, IBM Cloud, Ali Baba, Openshift, etc. It's "just" another platform Istio can run on.

File: content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md (hunk 2 above)
Comment (loewenstein) [reply content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md]: Done

File: content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md (hunk 1 above)
Comment (loewenstein) [reply content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md]: Done

**Pull request merged**